<?xml version="1.0"?>

<launch>
	<node respawn="true" pkg="rovitis_phidgets" type="phidgets_spatial" name="phidgets_spatial" output="log">
		<!--param name="dev_serial_number" type="int" value="301820" />-->
		<param name="dev_serial_number" type="int" value="301820" />
		<param name="dev_serial_label" type="string" value="phidgets_spatial" />
		<param name="frame" type="string" value="/phidgets_spatial" />
		<param name="wait_for_dev_delay" type="int" value="15000" />
		<param name="publish_frequency" type="int" value="5" />
		<param name="topic_name" type="string" value="/phidgets_spatial" />
		<param name="set_pose_service_name" type="string" value="phidgets_spatial_set" />
	</node>
</launch>
